src/main/java/org/opensearch/knn/common/KNNConstants.java

```diff
@@ -68,6 +68,8 @@ public class KNNConstants {
     public static final VectorDataType DEFAULT_VECTOR_DATA_TYPE_FIELD = VectorDataType.FLOAT;
 
     public static final String RADIAL_SEARCH_KEY = "radial_search";
+    public static final String INVALID_CODE_COUNT_ERROR_MESSAGE =
```

Comment:
We can add PQ word in the constant to be clear, like `INVALID_PQ_CODE_COUNT_ERROR_MESSAGE`.